src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultDocumentVersionCache.cs

```diff
@@ -15,7 +14,7 @@ internal class DefaultDocumentVersionCache : DocumentVersionCache
     internal const int MaxDocumentTrackingCount = 20;
 
     // Internal for testing
-    internal readonly Dictionary<DocumentKey, List<DocumentEntry>> DocumentLookup_NeedsLock;
+    internal readonly Dictionary<string, List<DocumentEntry>> DocumentLookup_NeedsLock;
```

Comment:
Unrelated to your change, it terrifies me a bit when I see an identifier containing the words "NeedsLock" but also "Internal for testing". :smile:

Reply:
Yea...